code/modules/reagents/reagent_dispenser.dm

```diff
@@ -109,6 +125,9 @@
 	..() //extend the zap
 	boom()
 
+/obj/structure/reagent_dispensers/attackby(obj/item/weapon/W, mob/user, params)
+	return
+

```

Comment:
Turns out I was wrong about this... these dispensors are so old that they let things handle filling them.